src/grafana_dashboards/dashboard.json

```diff
@@ -1426,7 +1426,7 @@
   },
   "timepicker": {},
   "timezone": "browser",
-  "title": "Charmed Opensearch Dashboards",
+  "title": "Charmed Wazuh Dashboard",
   "uid": "w7fRRIPVz",
```

Comment:
Should the `uid` not change? I don't really know, just asking :D

Reply:
In fact, it should be autogenerated, so I'm removing it. Thanks